tests/MatcherTest.php

```diff
@@ -258,7 +258,6 @@ public function testLogger()
         $matcher->match($request);
 
         $expect = [
-            'debug: /bar FAILED Aura\Router\Rule\Path ON foo',
```

Comment:
This will not work anymore because the /foo route will not be checked anymore.
Is this a problem for you? Do it really need hundreds of debug messages that a specific route didn't matched?